taskcluster/configs/config.ci.yml

```diff
@@ -29,36 +32,79 @@ experiment:
 
 marian-args:
   training-backward:
-    disp-freq: "1"
-    save-freq: "5"
+    # Run training for 10 updates, and display 5 updates. Only validate and save the
+    # model once.
+    disp-freq: "2"
```

Comment:
@gregtatum I don't think it's a good idea to touch model architecture too much here since those parameters are also something we want to test in CI. Student and backward models are already small. I suggest we do only minor changes here. You can try using Marian `task: transformer-base` instead of `task: transformer-big` but the main delay appeared after integration of OpusTrainer and it's likely this issue: https://github.com/mozilla/firefox-translations-training/issues/455